10up/ElasticPress
# Add new filter to customize HTTP headers in Autosuggest request

### Description of the Change

This change adds a new filter to customize HTTP headers in the Autosuggest request. 

### Alternate Designs

### Benefits
Allow adding custom HTTP headers to authentication or other purposes.

### Possible Drawbacks

### Verification Process

1. Add the filter `add_filter( 'ep_autosuggest_http_headers', function() { return [ 'X-Header-EP' => 'ep-custom-header' ]; } );`;
2. Type in an autosuggest field to fire the request;
3. Check if the header `X-Header-EP` is sent in the request.

### Checklist:






### Applicable Issues

Closes: #2048 

### Changelog Entry

Added the filter ep_autosuggest_http_headers
